.commitlintrc.yml

```diff
@@ -0,0 +1,62 @@
+# Commitlint configuration.
+# See: https://github.com/conventional-changelog/commitlint/blob/master/docs/reference-rules.md
+---
+rules:
+  body-leading-blank:
+    - 2
+    - always
+  body-max-line-length:
+    - 2
+    - always
+    - 75
+  footer-leading-blank:
+    - 2
+    - always
+  header-max-length:
+    - 2
+    - always
+    - 75
+  scope-case:
+    - 2
+    - never
+    - - sentence-case
+      - start-case
```

Comment:
Based on these examples:
```
  'lower-case',    // default
  'upper-case',    // UPPERCASE
  'camel-case',    // camelCase
  'kebab-case',    // kebab-case
  'pascal-case',   // PascalCase
  'sentence-case', // Sentence case
  'snake-case',    // snake_case
  'start-case'     // Start Case
```

I would suggest we allow pascal-case and lower-case instead of sentence-case and start-case, because I think there should not be spaces in the scope.
Currently we often use the class name as prefix, this could become for example `fix(MavenSupport): ...`

Reply:
> I think there should not be spaces in the scope.

I agree, that's why I use `never` (instead of `always` previously) now.